scrapper.py

```diff
@@ -35,6 +36,14 @@ def find_articles(self):
         """
         Finds articles
         """
+        # get data from website
+        response = requests.get('http://www.vestnik.vsu.ru/content/lingvo/index_ru.asp',
```

Comment:
I see that your headers are duplicated, do you know any way to better track them?